contracts/simple_dex/lib.rs

```diff
@@ -373,14 +373,27 @@ mod simple_dex {
             let this = self.env().account_id();
             let balance_token_in = self.balance_of(token_in, this)?;
             let balance_token_out = self.balance_of(token_out, this)?;
-
             if min_amount_token_out.map_or(false, |x| balance_token_out < x) {
                 // throw early if we cannot support this swap anyway due to liquidity being too low
                 return Err(DexError::NotEnoughLiquidityOf(token_out));
             }
 
+            Self::_out_given_in(
+                amount_token_in,
+                balance_token_in,
+                balance_token_out,
+                self.swap_fee_percentage,
+            )
+        }
+
+        fn _out_given_in(
```

Comment:
Cab we move the comment with the formula from the public `fn out_given_in` to here?

Reply:
wont be a part of the interface docstring if moved

Reply:
OK, it's just annoying to jump between public to private methods when verifying it.